src/routes.ts

```diff
@@ -0,0 +1,51 @@
+import express, { Express } from "express";
+import axios from "axios";
+import { API_KEY } from "./info";
+import { checkCameraType } from "./cameras";
+import { reduceInfo } from "./photo";
+import { getHtml } from "./helper";
+
+export const router = express.Router();
+
+// Part 2
+
+router.get("/rovers", function (req, res) {
+
+    console.log("Getting rovers ...");
+    const URL = `https://api.nasa.gov/mars-photos/api/v1/rovers?api_key=${API_KEY}`;
+
+    axios.get(URL)
+    .then((response) => {
+        const result = response.data;
+        res.send(result);
+    })
+    .catch(error => {
+        console.log(error)
+    });
```

Comment:
Immediately calling a method on the previous result, `.get(...).then(...).catch(...)`, is called chaining, and in general we'd prefer to indent continuations of the chain, especially for promises! Makes it easier to see which bits are being run asynchronously.